Source/FullScreenPokemon.ts

```diff
@@ -8745,7 +8763,10 @@ module FullScreenPokemon {
         }
 
         /**
+         * Determines the scrollable directions.
          * 
+         * @param FSP
+         * @returns The direction(s) that are scrollable.
```

Comment:
Can you add a `@todo` here for strict typing the returned string to a new IScrollability? When TypeScript 1.8 is out of beta, we'll be able to use string literals as types. This would be `"both" | "horizontal" | "vertical" | "none"`.